include/cudaq/Optimizer/Dialect/Quake/QuakeOps.td

```diff
@@ -754,6 +754,23 @@ class TwoTargetOp<string mnemonic, list<Trait> traits = []> :
 // Quantum operators (gates)
 //===----------------------------------------------------------------------===//
 
+def ExpPauliOp : QuakeOp<"exp_pauli", []> {
```

Comment:
The new ops and types should be added to the roundtrip.qke test.

Reply:
Will do.